src/nu/validator/checker/schematronequiv/Assertions.java

```diff
@@ -1734,6 +1734,16 @@ public void startElement(String uri, String localName, String name,
             warn("Element \u201Cmath\u201D does not need a"
                     + " \u201Crole\u201D attribute.");
         }
+        if ("http://www.w3.org/2000/svg" == uri) {
+            if ("a" == localName) {
```

Comment:
Should the `if`s be combined?

Reply:
Alright. I have updated this condition.

Reply:
Sorry if this was too nit picky, I'm not sure if the original was in line with the prevailing style of the codebase, just stuck out for me

Reply:
Actually your suggested change is more in style of the codebase. I have initially separated those conditions, because I expect more assertions be added for SVG.